src/components/Match/CastTable.jsx

```diff
@@ -34,19 +34,19 @@ const castsColumns = strings => [{
   displayFn: (row, col, field) => field || '-',
 }];
 
-const getCastArray = (pm) => {
+const getCastArray = (player) => {
   // Get from ability_uses, item_uses
   const resultArray = [];
   const targets = ['ability_uses', 'item_uses'];
```

Comment:
Should we also rename this? targets isn't really describing what this is used for. Should be called `castKeys` or something I guess?

Reply:
targetedCastTypes? kinda long but describes it well

Reply:
though I'm also okay with castKeys

Reply:
I'm fine with longer names as long as it makes sense and is easy to understand :)